Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -2406,14 +2407,13 @@ void MolDraw2D::extractAtomNotes(const ROMol &mol) {
     std::string note;
     if (atom->getPropIfPresent(common_properties::atomNote, note)) {
       if (!note.empty()) {
-        auto note_rect = calcAnnotationPosition(mol, atom, note);
-        if (note_rect.width_ < 0.0) {
+        AnnotationType annot;
+        annot.text_ = note;
+        calcAnnotationPosition(mol, atom, annot);
+        if (annot.rect_.width_ < 0.0) {
           cerr << "Couldn't find good place for note " << note << " for atom "
                << atom->getIdx() << endl;
```

Comment:
I know this isn't part of your PR, but as long as you're in here, please change this to:
```suggestion
          BOOST_LOG(rdWarningLog) << "Couldn't find good place for note " << note << " for atom "
               << atom->getIdx() << std::endl;
```